requirements/runtime.txt

```diff
@@ -11,4 +11,7 @@ pytorchcv==0.0.55
 tqdm
 onnx==1.6.0
 onnxruntime==1.1.2
-lxml
+lxml==4.5.0
+pycocotools==2.0.0
```

Comment:
This pip package is unofficial and kinda unstable. It should be better installed from the git repo:
```bash
pip install "git+https://github.com/cocodataset/cocoapi.git#subdirectory=PythonAPI"
```

Reply:
Replaced to 'pycocotools @ git+https://github.com/cocodataset/cocoapi.git#subdirectory=PythonAPI'
Is ok?

Reply:
I believe that `pycocotools` is a required dependency. Have you tried running training/testing without it installed?

Reply:
Yes,
```
Traceback (most recent call last):
  File "/mmdetection/tools/train.py", line 13, in <module>
    from mmdet.apis import get_root_logger, set_random_seed, train_detector
  File "/mmdetection/mmdet/apis/__init__.py", line 1, in <module>
    from .inference import (async_inference_detector, inference_detector,
  File "/mmdetection/mmdet/apis/inference.py", line 6, in <module>
    import pycocotools.mask as maskUtils
ModuleNotFoundError: No module named 'pycocotools'
```

Reply:
So, let's move it to `runtime` requirements.

Reply:
:rage:

Reply:
:+1:

Reply:
:tada: